cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsmachines.yaml

```diff
@@ -18,477 +18,6 @@ spec:
     singular: awsmachine
   scope: Namespaced
   versions:
-  - additionalPrinterColumns:
```

Comment:
why is all this being removed?

Reply:
talking with Cesar, I've recovered the `capi_types.go` and I've included it in the `support/api` module, now they're back again.